components/link/link.js

```diff
@@ -115,6 +115,7 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 				:host([lines]) a {
 					align-items: baseline;
 					display: flex;
+					min-width: 0;
```

Comment:
This is what overrides the default flex behavior of `min-width: auto` (which is essentially `fit-content`)